### ISSUE TITLE
Feature: Two VU meter bars if stereo, one if mono

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#710]
Subtle way of showing the user that Force Mono is on.
Could also just have a [MONO] thing somewhere.

**Comment by mpogue2** [comment]
This would also be useful for split tracks, like rounds, to see what is one which channel (L vs R).

**Comment by mpogue2** [comment]
Implemented by c76cdf999665d158909534ffba5aa8baf3efa31e .  I probably could have done this more efficiently by creating a struct containing L_mono, R, and isMono, but I was lazy, and just plumbed it the simplest way possible.  Works really nicely!

If mono, VU meter shows one bar.  If stereo, shows two independent bars, for L (top) and R (bottom).

I also optimized by calling update() only when one of the three parameters that affects the on-screen view of the VU meter has changed.  The VU meter seems quite responsive, and the CPU utilization while playing is about 15% of one core on M1 MacBook Air, which I think is acceptable.
